packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,41 @@
+import { IComboBoxProps } from '@fluentui/react/lib/ComboBox';
+
+// Can only be numbers between 0-23
+export interface TimeRange {
+  start: number;
+  end: number;
+}
+
+type PartialIComboBoxProps = Omit<IComboBoxProps, 'options'>;
+
+export interface ITimePickerProps extends PartialIComboBoxProps {
+  /**
+   * Label of the component
+   */
+  label?: string;
+  /**
+   * Time increments, in minutes, of the options in the dropdown
+   */
+  increments?: number;
+  /**
+   * Whether to show seconds in the component
+   */
+  showSeconds?: boolean;
+  /**
+   * Callback to localize the date strings displayed for dropdown options
+   */
+  onFormatDate?: (date: Date) => string;
+  /**
+   * Whether to use 12 or 24 hour time //
+   */
+  useHour12?: boolean;
+  /**
+   * Toggle to allow user input
+   */
+  allowFreeform?: boolean;
+  /**
+   * Custom time range to for time options
+   */
+  // TODO: Decide how this should be handled
```

Comment:
Is this something we want to handle in this PR or leave for the future?

Reply:
Zoinks, apologies for forgetting to remove this `TODO` as we have already decided on *a* way to handle time ranges. What are your thoughts on using interface 
```
export interface TimeRange {
  start: number;
  end: number;
}
```
where control users can provide start and end times, expressed as whole numbers between 0 and 23?